packages/server/lib/browsers/cdp_automation.ts

```diff
@@ -237,6 +238,8 @@ export class CdpAutomation implements CDPClient {
       headers: params.request.headers,
       resourceType: normalizeResourceType(params.type),
       originalResourceType: params.type,
+      cdpRequestWillBeSentTimestamp: params.wallTime * 1000,
```

Comment:
What's the precision of `wallTime`?

Reply:
https://vanilla.aslushnikov.com/?Network.TimeSinceEpoch

It's the time in seconds. Everything else is milliseconds, so I'm normalizing

Reply:
Added a comment: [`561448f` (#27976)](https://github.com/cypress-io/cypress/pull/27976/commits/561448fe5881fb3bb8de5f568730cd9b0973ebd7)